### ISSUE TITLE
This is not how you deal with logits

### ISSUE THREAD

**Issue by marcelklehr** [ntedgi/node-efficientnet#60]
**Describe the bug**
This algorithm produces the wrong outputs: https://github.com/ntedgi/node-efficientnet/blob/main/src/EfficientNetResult.ts#L26

**To Reproduce**
1. Set topK = 3
2. Let code predict something
3. All three labels are approximately 33%

**Expected behavior**
Get original model probabilities.

**Screenshots**
No.

**Additional context**
Google's mobilenet implementation illustrates how to deal with logits: https://github.com/tensorflow/tfjs-models/tree/master/mobilenet

**Comment by ntedgi** [comment]
which EfficientNetCheckPoint did you use?

**Comment by marcelklehr** [comment]
Ah, this may be due to me using Efficientnet v2 which churns out logits.
